src/main/core/work/task.rs

```diff
@@ -82,7 +82,9 @@ pub mod export {
     }
 
     impl CTaskHostTreePtrs {
-        /// SAFETY: Given that the host lock is held when execution of a
+        /// # Safety
+        ///
+        /// Given that the host lock is held when execution of a
```

Comment:
The column width is inconsistent with the following lines.

We're generally inconsistent with where we place the newlines in multi-row comments though, so maybe it's not worth fixing here. (Perhaps we need a lint for column width too? :)

Reply:
Fixed this one. I try to remember to reflow paragraphs, but yeah it's easy to miss, and rustfmt won't do it. :shrug:

It looks like rustfmt has [a feature for wrapping overly-long comment lines](https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=#wrap_comments), but I'm unclear whether it'd reflow "too short" lines like in this case. Also appears to be in [stabilization purgatory](https://github.com/rust-lang/rustfmt/issues/3347)